src/main/java/uk/gov/hmcts/reform/bulkscan/payment/processor/client/payhub/response/PaymentResult.java

```diff
@@ -0,0 +1,18 @@
+package uk.gov.hmcts.reform.bulkscan.payment.processor.client.payhub.response;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.List;
+
+public class PaymentResult {
+
+    public final List<String> paymentDcns;
+
+
```

Comment:
Does checkstyle not complain about the extra empty line?

Reply:
no fails but formatted it.